linkerd/app/inbound/src/http/tests.rs

```diff
@@ -58,7 +56,7 @@ async fn unmeshed_http1_hello_world() {
         version: proxy::http::Version::Http1,
         tcp: TcpAccept {
             target_addr: ([127, 0, 0, 1], 5550).into(),
-            client_addr: ([10, 0, 0, 41], 6894).into(),
+            client_addr: Remote(ClientAddr(([10, 0, 0, 41], 6894).into())),
```

Comment:
nit/TIOLI: seems like the `FooAddr` newtypes could have
```rust
impl<T> From<T> for FooAddr where T: Into<SocketAddr> { ... }
```
or something? but, this might make it more explicit, so :woman_shrugging:

Reply:
Yeah; I chose not to support From converters so that we actually have to write down what type it is. The converters let us fall back on implicit behavior. We can add these later if we get frustrated, but for now want to be explicit.

Reply:
ah, yup, that's valid, carry on!